app/views/Index.scala.html

```diff
@@ -62,7 +62,7 @@ <h2 class="app-task-list__section">
                 </li>
                 <li class="app-task-list__item">
                   <span class="app-task-list__task-name">
-                    <a href="@{controllers.routes.IndexController.onPageLoad().url}"aria-describedby="governing-doc-info-completed">
+                    <a href="@{controllers.regulatorsAndDocuments.routes.SelectGoverningDocumentController.onPageLoad(NormalMode).url}"aria-describedby="governing-doc-info-completed">
                       @messages("index.section2.spoke2.label")
```

Comment:
Same here

Reply:
This is temporary... we will have ticket to refactor task list codes.... that time everything will be normalised